Library/Homebrew/formula.rb

```diff
@@ -1063,6 +1063,18 @@ def std_cmake_args
     ]
   end
 
+  # Returns the path to a make 3.81 or newer binary.
+  # Returns the system make on Leopard and newer, and the
+  # path to the actually-installed make on Tiger or older.
+  def make_path
+    if MacOS.version > :tiger
```

Comment:
Wondering if this is worth detecting from `make --version` so e.g. it may help Linuxbrew too? Feel free to ignore that, though.